spec/system/case_contacts/index_spec.rb

```diff
@@ -1,28 +1,41 @@
 require "rails_helper"
 
 RSpec.describe "case_contacts/index", :disable_bullet, type: :system do
-  let(:organization) { create(:casa_org) }
   let(:volunteer) { create(:volunteer, display_name: "Bob Loblaw", casa_org: organization) }
-  let(:casa_case) { create(:casa_case, casa_org: organization, case_number: "CINA-1") }
-  let!(:case_assignment) { create(:case_assignment, volunteer: volunteer, casa_case: casa_case) }
-  let!(:case_contact) { create(:case_contact, creator: volunteer, casa_case: casa_case) }
+  let(:organization) { create(:casa_org) }
```

Comment:
I think it'll be more readable if you move this line back up

Reply:
because `organization` is currently referenced one line above where it was declared